cypress/e2e/5-ndv.cy.ts

```diff
@@ -511,6 +511,7 @@ describe('NDV', () => {
 		ndv.getters.outputDisplayMode().find('label').eq(1).should('include.text', 'JSON');
 		ndv.getters.outputDisplayMode().find('label').eq(1).click();
 
+		cy.wait(100);
```

Comment:
Should this wait for the `outputDataContainer` to be visible instead?

Reply:
Looks like this needs to be fixed properly in `master` to make this test less flaky.